projects/1.16/assets/apotheosis/apotheosis/lang/zh_cn.json

```
@@ -382,6 +382,18 @@
   "enchantment.apotheosis.magic_protection.desc": "减少玩家造成的魔法伤害，同时减少玩家受到的魔法伤害。",
   "enchantment.apotheosis.crescendo": "快速射击",
   "enchantment.apotheosis.crescendo.desc": "增加你的弩在装填完箭后瞄准时的射击次数。",
+  "enchantment.apotheosis.spearfishing": "捕鱼矛",

```

原文`Spearfishng`
这是个附魔，我不认为`捕鱼矛`的翻译足够好
```suggestion
  "enchantment.apotheosis.spearfishing": "捕鱼之矛",
```
```suggestion
  "enchantment.apotheosis.spearfishing": "鱼叉",
```
```suggestion
  "enchantment.apotheosis.spearfishing": "鱼获之矛",
```

---

这个我在翻译的时候完全忘了有鱼叉这么个词

---

这个我在翻译的时候完全忘了有鱼叉这个词